<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPointCloud3DSymbolWidget</class>
 <widget class="QWidget" name="QgsPointCloud3DSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>728</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="mLayerRenderingGroupBox_2">
     <property name="title">
      <string>Point Symbol</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="labelPointSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Point size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelMaxScreenError">
          <property name="text">
           <string>Maximum screen space error</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QgsDoubleSpinBox" name="mPointSizeSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPointBudget">
          <property name="text">
           <string>Point budget</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="mPointBudgetSpinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>5000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QgsDoubleSpinBox" name="mMaxScreenErrorSpinBox">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QgsCollapsibleGroupBox" name="mTriangulateGroupBox">
        <property name="title">
         <string>Render as a Surface (Triangulate)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="mHorizontalTriangleCheckBox">
           <property name="text">
            <string>Skip triangles longer than</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QgsDoubleSpinBox" name="mHorizontalTriangleThresholdSpinBox">
           <property name="toolTip">
            <string>Maximum Triangle Side Size in Horizontal Plan</string>
           </property>
           <property name="maximum">
            <double>10000000000000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="mVerticalTriangleCheckBox">
           <property name="text">
            <string>Skip triangles taller than</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QgsDoubleSpinBox" name="mVerticalTriangleThresholdSpinBox">
           <property name="toolTip">
            <string>Maximum Triangle Side 3D Height</string>
           </property>
           <property name="maximum">
            <double>10000000000000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowBoundingBoxesCheckBox">
        <property name="text">
         <string>Show bounding boxes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsCollapsibleGroupBox" name="mVpcGroupBox">
        <property name="title">
         <string>Virtual Point Cloud Options</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="mZoomOutOptions"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>When zoomed out</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QgsStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="noRendererPage"/>
     <widget class="QWidget" name="follow2DRendererPage">
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>See 2D Symbology settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="singleColorRendererPage">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QgsColorButton" name="mSingleColorBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="colorRampRendererPage">
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1" columnstretch="0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Attribute</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsPointCloudAttributeComboBox" name="mRenderingParameterComboBox"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QgsColorRampShaderWidget" name="mColorRampShaderWidget" native="true"/>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsDoubleSpinBox" name="mColorRampShaderMinEdit">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-9999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsDoubleSpinBox" name="mColorRampShaderMaxEdit">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-9999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mScalarRecalculateMinMaxButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rgbRendererPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="6" column="0">
          <widget class="QLabel" name="mContrastEnhancementAlgorithmLabel">
           <property name="text">
            <string>Contrast
enhancement</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="mRedBandLabel">
           <property name="text">
            <string>Red band</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="4">
          <widget class="QComboBox" name="mContrastEnhancementAlgorithmComboBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="mRedMinLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="mRedMaxLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="mRedMinLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QgsPointCloudAttributeComboBox" name="mRedAttributeComboBox"/>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="mBlueMinLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="mBlueMaxLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QgsPointCloudAttributeComboBox" name="mGreenAttributeComboBox"/>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="mGreenMinLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="mGreenMaxLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QgsPointCloudAttributeComboBox" name="mBlueAttributeComboBox"/>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="mRedMaxLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="mGreenMinLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="mBlueBandLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Blue band</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="mBlueMinLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLineEdit" name="mBlueMaxLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mGreenBandLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Green band</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLineEdit" name="mGreenMaxLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="classifiedrenderingPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="mClassifiedRenderingLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="mRenderingStyleComboBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsPointCloudAttributeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgspointcloudattributecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>qgsstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mRenderingStyleComboBox</tabstop>
  <tabstop>mSingleColorBtn</tabstop>
  <tabstop>mPointSizeSpinBox</tabstop>
  <tabstop>mMaxScreenErrorSpinBox</tabstop>
  <tabstop>mPointBudgetSpinBox</tabstop>
  <tabstop>mShowBoundingBoxesCheckBox</tabstop>
  <tabstop>mRenderingParameterComboBox</tabstop>
  <tabstop>mColorRampShaderMinEdit</tabstop>
  <tabstop>mColorRampShaderMaxEdit</tabstop>
  <tabstop>mScalarRecalculateMinMaxButton</tabstop>
  <tabstop>mContrastEnhancementAlgorithmComboBox</tabstop>
  <tabstop>mRedMinLineEdit</tabstop>
  <tabstop>mRedAttributeComboBox</tabstop>
  <tabstop>mGreenAttributeComboBox</tabstop>
  <tabstop>mBlueAttributeComboBox</tabstop>
  <tabstop>mRedMaxLineEdit</tabstop>
  <tabstop>mGreenMinLineEdit</tabstop>
  <tabstop>mBlueMinLineEdit</tabstop>
  <tabstop>mBlueMaxLineEdit</tabstop>
  <tabstop>mGreenMaxLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
